### ISSUE TITLE
serenity-rest XML/SOAP message content is not escaped in the html reports

### ISSUE THREAD

**Issue by erikgk** [serenity-bdd/serenity-core#88]
In the serenity reports the REST Query display does not escape xml / soap content. The browser therefore cannot display it correctly within the parent pre tag. 

![image](https://cloud.githubusercontent.com/assets/13133185/8452017/f66b933a-1fe7-11e5-9ff7-987cc4938100.png)

Example request:

```xml
<soapenv:envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsd="http://something/xsd">
  <soapenv:header><wsse:security soapenv:mustunderstand="0" xmlns:wsse="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-secext-1.0.xsd">
    <wsse:usernametoken>  
      <wsse:username>nonofyourbusiness</wsse:username> 
      <wsse:password>this also non of your business</wsse:password>
    </wsse:usernametoken>
  </wsse:security>
</soapenv:header>   
<soapenv:body>
  <xsd:namespacething>          
    <xsd:sometext>     
     <xsd:whatevernode>Whatevervalue</xsd:whatevernode>    
    </xsd:sometext> 
  </xsd:namespacething> 
</soapenv:body>
</soapenv:envelope>
```

**Comment by YamStranger** [comment]
Hello @erikgk! Thansk for creating this issue. 

I can't reproduce it, below you will find details. It seems that this issue is the same as #128 and already fixed and available under [v1.1.22-rc.4](https://github.com/serenity-bdd/serenity-core/releases/tag/v1.1.22-rc.4). Issue was in incorrect parsing Content type. If this issue can't be reproduces on new version of serenity bdd please close it.

If you have additional enhancements you can create additional issues as well.  

With current version your report should be like:

**Comment by erikgk** [comment]
We have concluded it is indeed already fixed. So #128 is the duplicate of this issue, but it's fixed either way.
